commands/smurf.js

```diff
@@ -0,0 +1,74 @@
+'use strict'
+// import jssha
+const jsSHA = require("jssha")
+
+const {
+  DEFAULT_SCHEMAS,
+  validate
+} = require('./../validation.js')
+
+// dec2hex
+function dec2hex(s) {
+  return (s < 15.5 ? "0" : "") + Math.round(s).toString(16)
+}
+// hex2dec
+function hex2dec(s) {
+  return parseInt(s, 16)
+}
+// leftpad
+function leftpad(str, len, pad) {
+  if (len + 1 >= str.length) {
+    str = Array(len + 1 - str.length).join(pad) + str
+  }
+  return str
+}
```

Comment:
You can use `String.padStart()` instead of this.

Reply:
now uses padStart